<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite skipped="0" hostname="DESKTOP-4J1LCJ7" name="com.w2a.suite.bankmanager.testcases.OpenAccountTest" tests="3" failures="3" timestamp="11 Apr 2019 15:22:24 GMT" time="375.125" errors="0">
  <testcase name="openAccountTest" time="163.117" classname="com.w2a.suite.bankmanager.testcases.OpenAccountTest">
    <failure type="java.lang.AssertionError" message="Failing while clicking on an Elementprocess_CSS">
      <![CDATA[java.lang.AssertionError: Failing while clicking on an Elementprocess_CSS
	at org.testng.Assert.fail(Assert.java:93)
	at com.w2a.datadriven.base.TestBase.reportFailure(TestBase.java:170)
	at com.w2a.datadriven.base.TestBase.click(TestBase.java:191)
	at com.w2a.suite.bankmanager.testcases.OpenAccountTest.openAccountTest(OpenAccountTest.java:31)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:108)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:661)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:869)
	at org.testng.internal.TestMethodWithDataProviderMethodWorker.call(TestMethodWithDataProviderMethodWorker.java:75)
	at org.testng.internal.TestMethodWithDataProviderMethodWorker.call(TestMethodWithDataProviderMethodWorker.java:14)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
]]>
    </failure>
  </testcase> <!-- openAccountTest -->
  <testcase name="openAccountTest" time="102.909" classname="com.w2a.suite.bankmanager.testcases.OpenAccountTest">
    <failure type="java.lang.AssertionError" message="Failing while clicking on an Elementprocess_CSS">
      <![CDATA[java.lang.AssertionError: Failing while clicking on an Elementprocess_CSS
	at org.testng.Assert.fail(Assert.java:93)
	at com.w2a.datadriven.base.TestBase.reportFailure(TestBase.java:170)
	at com.w2a.datadriven.base.TestBase.click(TestBase.java:191)
	at com.w2a.suite.bankmanager.testcases.OpenAccountTest.openAccountTest(OpenAccountTest.java:31)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:108)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:661)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:869)
	at org.testng.internal.TestMethodWithDataProviderMethodWorker.call(TestMethodWithDataProviderMethodWorker.java:75)
	at org.testng.internal.TestMethodWithDataProviderMethodWorker.call(TestMethodWithDataProviderMethodWorker.java:14)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
]]>
    </failure>
  </testcase> <!-- openAccountTest -->
  <testcase name="openAccountTest" time="109.099" classname="com.w2a.suite.bankmanager.testcases.OpenAccountTest">
    <failure type="java.lang.AssertionError" message="Failing while clicking on an Elementprocess_CSS">
      <![CDATA[java.lang.AssertionError: Failing while clicking on an Elementprocess_CSS
	at org.testng.Assert.fail(Assert.java:93)
	at com.w2a.datadriven.base.TestBase.reportFailure(TestBase.java:170)
	at com.w2a.datadriven.base.TestBase.click(TestBase.java:191)
	at com.w2a.suite.bankmanager.testcases.OpenAccountTest.openAccountTest(OpenAccountTest.java:31)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:108)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:661)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:869)
	at org.testng.internal.TestMethodWithDataProviderMethodWorker.call(TestMethodWithDataProviderMethodWorker.java:75)
	at org.testng.internal.TestMethodWithDataProviderMethodWorker.call(TestMethodWithDataProviderMethodWorker.java:14)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
]]>
    </failure>
  </testcase> <!-- openAccountTest -->
</testsuite> <!-- com.w2a.suite.bankmanager.testcases.OpenAccountTest -->
